tox.ini

```
@@ -139,7 +139,7 @@ deps=
 basepython=python3.6
 deps=
     Django>=2.2,<3.0
-    wagtail>=2.3,<2.10
+    -r{toxinidir}/requirements/wagtail-future.txt
```

Something like this is what I meant to suggest:

```suggestion
    wagtail==2.10rc2
    wagtail-autocomplete==0.6
    git+https://github.com/cfpb/wagtail-flags.git@wag210-test
    git+https://github.com/cfpb/wagtail-inventory.git@wag210-test
    git+https://github.com/cfpb/wagtail-sharing.git@wag210-test
    git+https://github.com/cfpb/wagtail-treemodeladmin.git@wag210-test
```

I'd really rather pin the temporary things we need for upgrading *here*, in the tox.ini, than in requirements files.

---

I can expand on this a little bit — I'm struggling with visualizing the "future" env a lot of the time — I think it might be worth talking about whether we go back to tox conventions for env naming for everything that's *not* `current` (`unittest-dj22-wag210`, for example). I don't want to make `-future` that little more concrete with a requirements file right now.

---

Changes are made as requested to remove requirements file wagtail-future.txt
Added temporary wagtail repos for upgrading in tox.ini